src/desktop/apps/auction/components/layout/meta.jade

```diff
@@ -7,7 +7,7 @@ meta( name='description', content= description )
 meta( property='og:description', content= description )
 meta( property='twitter:description', content= description )
 meta( property='og:url', content='#{sd.APP_URL}#{app.auction.href()}' )
-link( rel='canonical', href='#{sd.APP_URL}#{app.auction.href()}' )
+link( rel='canonical', href='#{sd.APP_URL}/auction/#{app.auction.id}' )
```

Comment:
@sweir27 ☝️ are you saying the canonical should be `sale` not `auction`?

Reply:
I think `auction` is good!

Is the `id` here the slug or the BSON id?

Reply:
I tested this locally and it was a slug 👍